spanner/google/cloud/spanner/streamed.py

```diff
@@ -61,7 +59,7 @@ def rows(self):
     def fields(self):
         """Field descriptors for result set columns.
 
-        :rtype: list of :class:`~google.cloud.proto.spanner.v1.type_pb2.Field`
+        :rtype: list of :class:`~google.cloud.proto.spanner.v1.types.Field`
```

Comment:
This seems inconsistent with the changes above, which use `google.cloud.spanner_v1.types`.

Reply:
Yep, this is wrong. Thanks.